src/signature: get key passphrase from environment (#945)
rauc/rauc: docs/advanced.rst

Note that you can specify the ``--intermediate`` argument multiple times to
include multiple intermediate certificates to your bundle signature.

Passphrase Handling
~~~~~~~~~~~~~~~~~~~

> If the signing key is protected by a passphrase it has to entered

Reviewer: ```suggestion
If the signing key is protected by a passphrase it has to be entered
```

> If the signing key is protected by a passphrase it has to entered
> during signing operation. For automation purpose the passphrase can be
> set in the environment variable RAUC_KEY_PASSPHRASE.

Reviewer 2: ```suggestion
If the signing key is protected by a passphrase it has to entered
during signing operations.
For automation purposes the passphrase can be
set using the environment variable RAUC_KEY_PASSPHRASE.
```

> environment variable RAUC_KEY_PASSPHRASE.

Reviewer: ```suggestion
environment variable ``RAUC_KEY_PASSPHRASE``.
```

.. note::

>   Since the environment of other processes is visible on Linux this

Reviewer: ```suggestion
  Since the environment of other processes is visible on Linux, this
```

>   used with caution.

Reviewer 2: We only support Linux.

Author: That was copied from the openssl man page :) fixed
